readthedocs/templates/projects/domain_list.html

```diff
@@ -12,11 +12,22 @@
 {% block project_edit_content_header %}{% trans "Domains" %}{% endblock %}
 
 {% block project_edit_content %}
-  <p class="help_text">
-    {% blocktrans trimmed with docs_url='https://docs.readthedocs.io/page/custom_domains.html' %}
-        Configuring a custom domain allows you to serve your documentation from a domain other than "{{ default_domain }}". <a href="{{ docs_url }}">Learn more</a>.
-    {% endblocktrans %}
-  </p>
+  {% if superproject %}
+    {% url "projects_detail" superproject.slug as superproject_url %}
+    <p class="help_text">
```

Comment:
don't think we have something like a class for a "warning" banner.

Reply:
We do not, not yet at least!